### DISCUSSION TITLE
Remote Borg Backup

### DISCUSSION THREAD

**Discussion by msullivan-lc** [nextcloud/all-in-one#5681] [Questions] [answered]
Now that  #4804 is complete and merged, is there an easy way to "reset" the backup system, and start fresh to the remote Borg repo?

Thanks,
Michael

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one/discussions/5593

**Reply by msullivan-lc** [reply]
Wow...! I swear I searched! Thank you kindly.
